### DISCUSSION TITLE
When I access http://127.0.0.1:7091, error 404 is reported

### DISCUSSION THREAD

**Discussion by YvCeung** [apache/incubator-seata#6962] [Q&A] [answered]
I successfully started seata-server in local IDEA (Figure 1), but when I access the seata console UI, it cannot be opened(Figure 2).
However, the health check interface is also ok (Figure 3)
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/b62c9b97-a41a-41a1-98b6-5dadede7faa2">
<img width="983" alt="image" src="https://github.com/user-attachments/assets/cc7bc95f-701b-426d-9515-4d7ef8df11df">
<img width="897" alt="image" src="https://github.com/user-attachments/assets/29f0958c-d37b-483c-ad5c-dc88689f6443">

why?

**Comment by psxjoy** [comment]
Did you change any local configurations? 
I just pulled the latest 2.x code, and the project started up fine, automatically redirecting to the [login page](http://127.0.0.1:7091/#/login).
(Consider running `mvn clean` or restarting the IDE to clear any possible compilation or dependency cache issues—could that help?）

**Comment by funky-eyes** [comment, marked as answer]
In version 2.4, the console was moved to the naming server, so the server on port 7091 is only used for health checks.
